### DISCUSSION TITLE
Quiero saber si puedo obtener la cuenta Github desde Uruguay...

### DISCUSSION THREAD

**Discussion by Crissper1** [community/community#134104] [GitHub Education]
### Hi everyone,

Hola comunidad quería saber si es posible obtener la cuenta de estudiante desde Uruguay
Estoy en el curso de bachillerato tecnológico. En la UTU Brazo Oriental en Montevideo...

Gracias desde ya.

PD: Ya eh intentado la autorización pero me han rechazado...
Teniendo una carta de la UTU...

**Comment by Jesusavvillalba** [comment]
Verifica los Requisitos: Asegúrate de cumplir con todos los requisitos necesarios para obtener la cuenta de estudiante. Algunas plataformas o servicios pueden tener criterios específicos que debes cumplir.

Contacta Soporte Técnico: Muchas plataformas tienen un equipo de soporte que puede proporcionar orientación adicional. Explica tu situación y proporciona cualquier documento de respaldo.

Consulta con tu Institución: Habla con el personal administrativo de la UTU para ver si pueden ofrecerte alguna carta adicional o documentación que pueda fortalecer tu solicitud.

Explora Alternativas: Si sigues teniendo dificultades, podrías buscar alternativas a la cuenta de estudiante que estás tratando de obtener. A veces, hay otros programas o beneficios disponibles para estudiantes que podrían ser útiles.

**Comment by davevad93** [comment]
Hi @Crissper1 ,

Yes you can apply from Uruguay, you just need to fulfill some requirements one of which is to provide your academic documentation as proof; here's the link to the GitHub docs in Spanish on how to apply to GitHub Education as a student:

- [Envío de solicitud a GitHub Education como estudiante](https://docs.github.com/es/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-education-for-students/apply-to-github-education-as-a-studen)

**Comment by juanpflores** [comment]
Hola @Crissper1 👋🏼 
Revise tu applicacion a los beneficios del programa. Por favor revisa tu correo en los proximos 3 dias

**Comment by Crissper1** [comment]
> ### Hi everyone,
> 
> Hola comunidad quería saber si es posible obtener la cuenta de estudiante desde Uruguay
> Estoy en el curso de bachillerato tecnológico. En la UTU Brazo Oriental en Montevideo...
> 
> Gracias desde ya.
> 
> PD: Ya eh intentado la autorización pero me han rechazado...
> Teniendo una carta de la UTU...

Gracias a todos por su respuesta. 
Cierro hilo. 
Ya me dieron la autorización
